### DISCUSSION TITLE
【任务（已完成）】“审核+图片补充”计划 2｜5,000 $PEOPLE

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#96] [任务 | Bounty]
WTFSolidity极简教程-应用篇前10讲（总 31-40讲），但仍然没有经过严格的编辑审核，有的部分也少一些概念的示意图或程序运行的截图，不利于新手学习。
为了让WTFSolidity极简教程更好，我发起“审核+图片补充”计划，每一讲由1个社区成员认领，负责审核校对，并添加一些图片。
**赏金**：5,000 $PEOPLE 代币 + 7月WTF社群奖励，根据工作量分配。
在这个帖子下面回复就能认领（目前1人最多认领2讲，完成后可认领新的）。

有用的资料：
1. Markdown书写规则：https://www.markdownguide.org/cheat-sheet/
2. 审核并添加图片后的例子：https://github.com/AmazingAng/WTFSolidity/tree/main/02_ValueTypes
3. 审核第一期：https://github.com/AmazingAng/WTFSolidity/discussions/18

### 认领情况：
第31讲：ERC20
@buttonwild  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/99
第32讲：代币水龙头
@Hongchenglong   **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/100
第33讲：空投
@Hongchenglong   **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/100
第34讲：ERC721
@buttonwild  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/104
第35讲：荷兰拍卖
@JustinAsdz **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/108 https://github.com/AmazingAng/WTFSolidity/pull/110
第36讲：Merkle Tree
@JustinAsdz   **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/102
第37讲：数字签名
@sdfgsdfsdf  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/109
第38讲：NFT交易所
@EasyChris  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/107
第39讲：随机数（链上/chainlink）
@zhiyuan2007 **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/112
第40讲：ERC1155
@reborn-sama  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/111

奖励已经发放，感谢各位贡献者！

tx: https://etherscan.io/tx/0x70b299b79743d5741f7fbed891df0258e33ce035fdf950b295533395e08c9b50

**Comment by Hongchenglong** [comment]
32、33

**Comment by JustinAsdz** [comment]
35，36

**Reply by AmazingAng** [reply]
35讲需要加些remix验证图片

**Comment by buttonwild** [comment]
31,34

**Comment by sdfgsdfsdf** [comment]
37

**Reply by AmazingAng** [reply]
什么时候可以审核好呢？

**Comment by reborn-sama** [comment]
40

**Reply by AmazingAng** [reply]
40讲已经更新

**Comment by EasyChris** [comment]
38

**Reply by AmazingAng** [reply]
需要在好好看下markdown，审核完认真读下语句是否通顺。
我做了些修改：
https://github.com/AmazingAng/WTFSolidity/commit/4f2ad560c8b5c44dddb0b3a7a5a76fe8f9f64b7f

也把除了`remix`实现部分的教程审核下吧，查下语病。

**Reply by EasyChris** [reply]
收到，之后会改进！感谢指正。

**Comment by zhiyuan2007** [comment]
39

**Reply by AmazingAng** [reply]
https://github.com/AmazingAng/WTFSolidity/tree/main/39_Random

**Reply by zhiyuan2007** [reply]
好的，收到

**Comment by sdfgsdfsdf** [comment]
收到



在 2022-07-19 11:29:23，"0xAA" ***@***.***> 写道：

需要在好好看下readme，审核完认真读下语句是否通顺。
我做了些修改：
4f2ad56

也把除了remix实现部分的教程审核下吧，查下语病。

**Comment by Tron933** [comment]
> WTFSolidity极简教程-应用篇前10讲（总 31-40讲），但仍然没有经过严格的编辑审核，有的部分也少一些概念的示意图或程序运行的截图，不利于新手学习。
> 为了让WTFSolidity极简教程更好，我发起“审核+图片补充”计划，每一讲由1个社区成员认领，负责审核校对，并添加一些图片。
> **赏金**：5,000 $PEOPLE 代币 + 7月WTF社群奖励，根据工作量分配。
> 在这个帖子下面回复就能认领（目前1人最多认领2讲，完成后可认领新的）。
> 
> 有用的资料：
> 1. Markdown书写规则：https://www.markdownguide.org/cheat-sheet/
> 2. 审核并添加图片后的例子：https://github.com/AmazingAng/WTFSolidity/tree/main/02_ValueTypes
> 3. 审核第一期：https://github.com/AmazingAng/WTFSolidity/discussions/18
> 
> ### 认领情况：
> 第31讲：ERC20
> @buttonwild  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/99
> 第32讲：代币水龙头
> @Hongchenglong   **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/100
> 第33讲：空投
> @Hongchenglong   **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/100
> 第34讲：ERC721
> @buttonwild  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/104
> 第35讲：荷兰拍卖
> @JustinAsdz **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/108 https://github.com/AmazingAng/WTFSolidity/pull/110
> 第36讲：Merkle Tree
> @JustinAsdz   **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/102
> 第37讲：数字签名
> @sdfgsdfsdf  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/109
> 第38讲：NFT交易所
> @EasyChris  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/107
> 第39讲：随机数（链上/chainlink）
> @zhiyuan2007 **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/112
> 第40讲：ERC1155
> @reborn-sama  **已完成** Execellent https://github.com/AmazingAng/WTFSolidity/pull/111
> 
> 奖励已经发放，感谢各位贡献者！
> 
> tx: https://etherscan.io/tx/0x70b299b79743d5741f7fbed891df0258e33ce035fdf950b295533395e08c9b50 
````
> 
>
